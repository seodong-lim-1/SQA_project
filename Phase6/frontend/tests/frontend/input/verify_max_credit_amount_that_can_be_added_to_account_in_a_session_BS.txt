login
testBSUser
password
addcredit
1500
720
logout
