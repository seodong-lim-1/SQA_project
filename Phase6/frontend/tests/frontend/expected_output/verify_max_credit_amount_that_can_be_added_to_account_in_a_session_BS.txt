Welcome to the Online Auction Sales Service!

Please enter login to Login.
login - Login

***LOGIN***
Enter Username: 
Enter Password: 
Welcome Buy-Standard User: testBSUser!

***MENU***
logout - Logout
bid - Make a bid on an available item
addcredit - Add Credit
listitems - List Items Up For Auction
***Add Credit***
User Found! Please proceed to add a credit amount.
Enter the amount of credit to add: Credit Amount to be added is greater than 1000. Please try again.

Enter the amount of credit to add: 

The specified credit has been successfully add to the indicated account!

NOTE: You've reached your session transaction limit. Please enter logout to log out and log back in to continue.

***MENU***
logout - Logout
bid - Make a bid on an available item
addcredit - Add Credit
listitems - List Items Up For Auction

You have successfully logged out.
